ml-proto/spec/memory.mli

```diff
@@ -4,11 +4,9 @@
 
 type memory
 type t = memory
-type address = int
```

Comment:
Specific reason to remove these instead of making them synonyms for int64? They have some documentation value.